config/config.go

```diff
@@ -1892,7 +1892,10 @@ func getParserConfig(name string, tbl *ast.Table) (*parsers.Config, error) {
 // a serializers.Serializer object, and creates it, which can then be added onto
 // an Output object.
 func buildSerializer(name string, tbl *ast.Table) (serializers.Serializer, error) {
-	c := &serializers.Config{TimestampUnits: time.Duration(1 * time.Second)}
+	c := &serializers.Config{
+		TimestampUnits:       time.Second,
```

Comment:
I don't think this guy is being used anywhere

Reply:
I wonder why they were doing time.Duration(1 * time.Second). Seems silly.

Reply:
Yeah, not sure why they did it that way.

Ben, it's used by the JSON serializer which is outside the scope of the diff.